inst/app/global.R

```diff
@@ -29,6 +29,6 @@ library(leaflet.extras)
 # Data import -------------------------------------------------------------
 
 ## Take data from {hkdatasets}
-hk_accidents <- hkdatasets::hk_accidents
+hk_accidents <- hkdatasets:::hk_accidents
```

Comment:
hk_accidents is currently not exported to bypass R CMD checks because of file size. See https://github.com/Hong-Kong-Districts-Info/hkdatasets/issues/15 for details.